public/src/js/widgets/columns/packages.js

```diff
@@ -23,17 +23,37 @@ export default class Package extends ColumnWidget {
         this.displayName = ko.observable();
         this.searchResults = ko.observableArray();
         this.searchedPackages = ko.observable();
+        this.editingPackage = ko.observable(null);
 
         this[bouncedSearch] = debounce(performSearch.bind(this), CONST.searchDebounceMs);
+
+        this.listenOn(mediator, 'package:edit', (packageId) => {
```

Comment:
The package is saved in the story-package widget as a Collection, which is the wrong format, but it is saved in the correct format in the baseModel.